e2e/init.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable flowtype/require-valid-file-annotation */
-/* globals jasmine beforeAll beforeEach afterAll */
+/* globals jasmine beforeAll afterAll */
```

Comment:
Remove this and add...

/* eslint-env detox/detox, jest */

Reply:
This threw a linting error when removed due to jasmine not being defined. This will be refactored away when we switch to jest circus